superduper/base/config.py

```diff
@@ -119,6 +119,28 @@ class CDCConfig(BaseConfig):
     )
 
 
+@dc.dataclass
+class QdrantConfig(BaseConfig):
+    """Describes the configuration for Qdrant.
+
+    :param location: The location for the Qdrant service.
+    :param prefer_grpc: Whether to prefer gRPC for the Qdrant service.
+    :param https: Whether to use HTTPS for the Qdrant service.
+    :param api_key: The API key for the Qdrant service.
+    :param prefix: The prefix for the Qdrant service.
+    :param timeout: The timeout for the Qdrant service.
+    :param vector_name: The name of the vector in the collection.
+    """
+
+    location: t.Optional[str] = ":memory:"
```

Comment:
This is fine, but since we might support many vector-search implementations, it would probably be good to include a more generic way to add parameters to the vector-search backend. 

For example, you could add `CFG.vector_search_kwargs` and then pass these on to the vector-search class.

Reply:
Did you have a look at this? Do you need help on that feature?

Reply:
Done in [`42683a8` (#2428)](https://github.com/superduper-io/superduper/pull/2428/commits/42683a87f93b6b31ed45e21e75874f3d633ec141)